studiopress/atomic-blocks
# Add migration admin page

This PR adds an admin submenu page called "Migrate" under the "Atomic Blocks" menu item. It contains information about migration, checks the PHP version to ensure it is 7.1 or higher, and provides a button to "Install Genesis Blocks".

See https://wpengine.atlassian.net/secure/RapidBoard.jspa?rapidView=441&projectKey=GF&modal=detail&selectedIssue=GF-2417

### How to test
1. In wp-admin, under "Atomic Blocks" click "Migrate".
2. Set your PHP version to something below PHP 7.1, and make sure you see the error telling you to upgrade.
3. Set your PHP version to something PHP 7.1 or newer, and make sure you see the "Install Genesis Blocks" button.

Note that this still requires hooking up the "Install Genesis Blocks" button to the flow described here:
https://wpengine.atlassian.net/browse/GF-2418

### Documentation
Migration page added for Genesis Blocks.

### Suggested changelog entry
Migration page added for Genesis Blocks.
